src/GridTable.js

```diff
@@ -117,13 +128,13 @@ class GridTable extends React.PureComponent {
           frozenData={frozenData}
           width={width}
           height={Math.max(height - headerHeight - frozenRowsHeight, 0)}
-          rowHeight={estimatedRowHeight ? getRowHeight : rowHeight}
+          rowHeight={(estimatedRowHeight || columnWidths) ? (getRowHeight ? getRowHeight : () => rowHeight)  : rowHeight}
```

Comment:
why does it need to be a function? it sems they all return rowHeight in the end?

Reply:
The signature is different in different modes.
Check out like 115 where selection is made between VariableSizeGrid and FixedSizeGrid